core/frontend/src/BackgroundMapGeometry.ts

```diff
@@ -357,7 +357,7 @@ export class BackgroundMapGeometry {
         }
 
         const ellipsoidColor = ColorDef.create(ColorByName.yellow);
-        builder.setSymbology(ellipsoidColor, ellipsoidColor, 1, 2);
+        builder.setSymbology(ellipsoidColor, ellipsoidColor, 1, 2 as LinePixels);
```

Comment:
Another suspect 2...

Reply:
Yeah, "2" is not a meaningful line pixel value. `Code2`is as good a choice as any; these are only debug graphics.

Reply:
```suggestion
        builder.setSymbology(ellipsoidColor, ellipsoidColor, 1, LinePixels.Code2);
```

Reply:
Gotta import LinePixels.

Reply:
wasn't the import already added for the cast?

Reply:
Yeah. derp.